modules/nw-multus-configuring-whereabouts-ip-reconciler-schedule.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="nw-multus-configuring-whereabouts-ip-reconciler-schedule_{context}"]
+= Configuring the Whereabouts IP reconciler schedule
+
+The Whereabouts IPAM CNI plugin runs the IP reconciler daily to clean up any stranded IP allocations that might result in exhausting IPs and therefore prevent new Pods from getting an IP allocated to them.
```

Comment:
Lowercase no-markup for [general reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references):
```suggestion
The Whereabouts IPAM CNI plugin runs the IP reconciler daily to clean up any stranded IP allocations that might result in exhausting IPs and therefore prevent new pods from getting an IP allocated to them.
```
Also seems a bit long of a sentence, but it does seem tricky to break up. Not critical, but, a crack at it:
```suggestion
The Whereabouts IPAM CNI plugin runs the IP reconciler daily to clean up any stranded IP allocations. Otherwise, IP pools could become exhausted and therefore prevent new pods from getting an IP allocated to them.
```
Not sure that that's any better... 😅